source/features/one-click-diff-options.tsx

```diff
@@ -1,68 +1,12 @@
 import React from 'dom-chef';
 import select from 'select-dom';
-import delegate, {DelegateEvent} from 'delegate-it';
 import * as pageDetect from 'github-url-detection';
 import {BookIcon, CheckIcon, DiffIcon, DiffModifiedIcon} from '@primer/octicons-react';
 
 import features from '../feature-manager';
-import attachElement from '../helpers/attach-element';
 import observe from '../helpers/selector-observer';
 import {removeTextNodeContaining} from '../helpers/dom-utils';
 
-const diffSwitchButtons = features.getIdentifiers(import.meta.url);
-
-function alternateDiffNatively(event: DelegateEvent<MouseEvent, HTMLAnchorElement>): void {
-	const type = new URLSearchParams(event.delegateTarget.search).get('diff')!;
-	const formField = select(`input#diff_${type}`);
-	if (!formField) {
-		// Let the link through
-		return;
-	}
-
-	// Submit form so that the preference is persisted #5288
-	formField.checked = true;
-	formField.form!.submit();
-	event.preventDefault();
-}
-
-function makeLink(type: string, icon: Element, selected: boolean): JSX.Element {
-	const url = new URL(location.href);
-	url.searchParams.set('diff', type);
-	const classes = pageDetect.isPR()
-		? 'ml-2 color-fg-muted'
-		: 'btn btn-sm BtnGroup-item ' + (selected ? 'selected' : '');
-
-	return (
-		<a
-			className={`tooltipped tooltipped-s ${classes} ${diffSwitchButtons.class}`}
-			aria-label={`Switch to the ${type} diff view`}
-			href={url.href}
-		>
-			{icon}
-		</a>
-	);
-}
-
-function createDiffStyleToggle(): DocumentFragment {
-	const isUnified = select.exists([
-		'[value="unified"][checked]', // Form in PR
-		'.table-of-contents .selected[href*="diff=unified"]', // Link in single commit
-	]);
-
-	if (pageDetect.isPR()) {
-		return isUnified
-			? makeLink('split', <BookIcon className="v-align-middle"/>, false)
-			: makeLink('unified', <DiffIcon className="v-align-middle"/>, false);
-	}
-
-	return (
-		<>
-			{makeLink('unified', <DiffIcon/>, isUnified)}
-			{makeLink('split', <BookIcon/>, !isUnified)}
-		</>
-	);
-}
```

Comment:
So much dead code here, we haven't used half of this code for who knows how long